scripts/sanitycheck

```diff
@@ -1015,12 +1015,19 @@ class MakeGenerator:
 
         args.append("BOARD={}".format(ti.platform.name))
         args.extend(extra_args)
-        if (ti.platform.qemu_support and (not ti.build_only) and
-                (not build_only) and (enable_slow or not ti.test.slow)):
+
+        _run_slow = enable_slow or not ti.test.slow
```

Comment:
curious about the _ prefix to these function local variables? Is that in the Python style PEP?

Reply:
fixed.